drawBot/context/tools/openType.py

```diff
@@ -470,12 +471,14 @@
     reversedFeatureMap[(featureType, featureSelector)] = key
 
 
+@memoize
 def getFeatureTagForFontAttribute(attribute):
     featureType = attribute.get(CoreText.NSFontFeatureTypeIdentifierKey)
     featureSelector = attribute.get(CoreText.NSFontFeatureSelectorIdentifierKey)
     return reversedFeatureMap.get((featureType, featureSelector))
 
 
+@memoize
 def getFeatureTagsForFontAttributes(attributes):
```

Comment:
This one (and the one above) are only used for svg export it seems, so should not influence your timing test at all.

Reply:
Also, I bet you didn't test this, because memoize arguments need to be hashable, which is not the case here, so this can't work.